### ISSUE TITLE
Cannot approve plan

### ISSUE THREAD

**Issue by ownercoder** [GalleyBytes/require-approval#1]
Docker image with "require-approval" tool has base image "scratch", in current configuration no way to simply approve/cancel plan by creating file in fs, because base image has no tools touch, cp, cat, ...

**Comment by isaaguilar** [comment]
That's true, the image doesn't allow for exec or anything like it. There are two ways approvals are done, 1st by observing a file and 2nd by observing a database entry. The latter is a different project which someday I might open it up. 

But back to the dilemma at hand. I would use the tfo-cli tool (https://github.com/isaaguilar/terraform-operator-cli/releases/tag/v2.1.1) and you can run a command like the following: 

```
tfo local debug --namespace <my-namespace> <my-terraform-resource> -- touch /home/tfo-runner/generations/<n>/_approved_<uuid-of-plan>
``` 

> Side note: 
>
> I want to add a native command to tfo-cli eventually. 
>
> _the following doesn't work, it's just an idea_
>
> ```
> tfo local approve -n <my-namespace> <my-terraform-resource>
> ```
